Global.h

```diff
@@ -753,6 +753,12 @@ namespace vstd
 		if(i < 0) return -i;
 		return i;
 	}
+
+	///C++23

```

Comment:
Since when do we support or require C++23?

Reply:
We does not support C++23, it is just a feature backported from c++23, in global there is many such features, like clamp from c++17.